sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -26,7 +26,7 @@
 class KeyClient(KeyVaultClientBase):
     """A high-level interface for managing a vault's keys.
 
-    :param str vault_url: URL of the vault the client will access
+    :param str vault_url: URL of the vault the client will access. In the Azure Portal, this is the vault's "DNS Name".
```

Comment:
Can we update this to "This is also referred to as the vault's DNS Name" rather than "In the portal" we have asked the portal team to change this so it will likely change in the near future.,